lib/builder.rb

```diff
@@ -418,7 +422,22 @@ def create_topic(user, action, data)
   end
 
   def send_message(user, action, data)
-    title = data[action['title']]
+
+    empty_data = false
```

Comment:
@merefield We can't do it this way unfortunately. It's too broad / catch-all, particularly considering we all fields to be optional. Outside of Jules' immediate use case, there will be other use cases where people still want to send a message if an optional field is not filled.  We need to:

1. Add a field to the send_message action: "Requires" being filled by ``wizard-custom-inputs`` selector, or just a combo-box with ``availableFields`` as the content.

2. Require the presence of a "Requires" field in the data.

Reply:
Can you check my changes please?

Reply:
I'm getting this exception when trying to submit the wizard *without* filling out the "required" field in the send_message action. This should be possible, i.e. the wizard should submit and the action should not completed.

<img width="689" alt="Screen Shot 2019-10-10 at 11 05 12 AM" src="https://user-images.githubusercontent.com/5931623/66529150-10012e00-eb4e-11e9-9655-356bf967b4f3.png">
<img width="701" alt="Screen Shot 2019-10-10 at 11 05 03 AM" src="https://user-images.githubusercontent.com/5931623/66529152-10012e00-eb4e-11e9-83b5-6f9745e77db1.png">

Reply:
@merefield Yup, it seems to be working for me too. Not sure what happened before.